client/src/index.tsx

```diff
@@ -7,12 +7,14 @@ import { Integrations } from '@sentry/tracing'
 import './index.css'
 import App from './App'
 
-Sentry.init({
-  dsn: (window as any)._arlo_sentry_dsn, // eslint-disable-line @typescript-eslint/no-explicit-any
-  environment: (window as any)._arlo_flask_env, // eslint-disable-line @typescript-eslint/no-explicit-any
-  integrations: [new Integrations.BrowserTracing()],
-  tracesSampleRate: 0.2,
-})
+if (process.env.NODE_ENV === 'production') {
```

Comment:
My understanding is that `process.env.NODE_ENV` gets replaced with either `"development"` or `"production"` depending on whether you're running the vite dev server or using a vite production build. @eventualbuddha does that sound right?

Reply:
Yes, but I can't remember whether that happens with vanilla `vite`. I think it does, because we're not specifically setting that `define` config in any of the vxsuite `vite` config.